pages/docs/concepts/asyncapi-document/structure.md

```diff
@@ -0,0 +1,433 @@
+---
+title: AsyncAPI document structure
+weight: 32
+---
+The structure of a AsyncAPI document can be defined as a specific format in which the document of the specification is to be written. An AsyncAPI document may be made up of a single document or be divided into multiple, connected parts at your discretion. In the latter case, Reference Objects are used. The structure of a AsyncAPI document has certain fields that need to be followed and implemented.
+This document lays out the blueprint for you to write down your specification based on your application. 
+
+## Root Elements 
+Root elements of an AsyncAPI document provide an overview of the API's characteristics and behavior. These root elements collectively define the structure, metadata, channels, components and more of a AsyncAPI document. They provide a comprehensive overview of the API's characteristics and behavior.
+
+```mermaid
+graph LR
```

Comment:
Redirect the diagrams to individual objects

Reply:
I think you can achieve it by:

```mermaid
graph LR
A[asyncapi]
B[<a href='#info-object'>info</a>]
C[<a href='#servers-object'>servers</a>]
D[<a href='#channels-object'>channels</a>]
E[<a href='#operations-object'>operations</a>]
F[<a href='#components-object'>components</a>]
A --> B
A --> C
A --> D
A --> E
A --> F
```

Reply:
oooooh fancy smancy..

Reply:
Looks like redirecting to individual sections of the document isn't possible here unless we use the href tag and reference to a link. I tried referencing the way you suggested and some other ways but it doesn't seem to work, since this was a nice to have for the document, I'm resolving the conversation. :)